pages/DSNP/Identifiers.md

```diff
@@ -49,11 +49,29 @@ Any [Announcement Types](Announcements.md#announcement-types) with a `fromId` an
 
 ### Example
 ```
-dsnp://78187493520/0x1234567890abcdef0123456789abcdef0123456789abcdef0123456789abcdef
+dsnp://78187493520/QmQNHNfHnbgJJ6nK4UPx2VtTUCafAKCbqZJ6ZRYUGjoeFj
 ```
 
 | part | value |
 | ---- | ----- |
 | Scheme | `dsnp://` |
 | User Id | `78187493520` |
-| Content Hash | `0x1234567890abcdef0123456789abcdef0123456789abcdef0123456789abcdef` |
+| Content Hash | `QmQNHNfHnbgJJ6nK4UPx2VtTUCafAKCbqZJ6ZRYUGjoeFj` |
```

Comment:
I believe this change should also propagate to `pages/ActivityContent/Associated/Hash.md` Or would that require https://github.com/LibertyDSNP/spec/issues/273 to be implemented first?

Reply:
This PR is just changing it to a multihash multibase string, as we don't serialize hashes as hex anywhere as far as I know. (The DSNP URI should match the hash from the announcement, which is now a multihash.)

Technically unrelated to the purpose of this PR, just a vestigial issue I happened to run into. I can make it a separate PR if you think it's warranted.

Reply:
It is fine. Just checking